ocw-course-v2/ocw-studio.yaml

```
@@ -185,6 +185,7 @@ collections:
           - "Online Textbook"
           - "Other Audio"
           - "Other Video"
+          - Podcast
```

I'm not sure why all these values are plural, but I think we should be consistent. 

```suggestion
          - Podcasts
```